content/topics/unit-testing/tdd-horrors/_index.md

```diff
@@ -0,0 +1,214 @@
+---
+title: TDD Horrors
+ready: true
+---
+
+# Common TDD pitfalls.
+These are all the most common TDD problems recruits tend to come across, read and make sure you didn't make any of these or any mistakes at all.
+
+- **Write tests**. Recruits in the past have done the mistake of not writing tests for TDD assignments, make sure you avoid this weird mistake. always write tests for your TDD projects.
+- No code in your TDD file, This is a repetition of the first point, but it's worth it. write tests people.
+- Files naming convention, name your files properly and according to what's inside them. E.g. if you have a file called add.js that only contains a multiply function then something is wrong. Do this:
+<!-- examples -->
+
+**Python**:
+
+`add.py`
```

Comment:
I'm worried that if we give them this advice they'll end up giving us a million little files, one for each function. Don't lead them to fall into new traps.  

how about 

```
calculator.py
# etc
```